security/container_security/security-container-signature.adoc

```diff
@@ -18,6 +18,17 @@ include::modules/containers-signature-verify-enable.adoc[leveloffset=+1]
 //Verifying that the enable signature verification is active
 include::modules/containers-signature-verify-application.adoc[leveloffset=+1]
 
+//Understanding unsigned container images within a release
+include::modules/containers-signature-verify-unsigned.adoc[leveloffset=+1]
+
+[id="additional-resources"]
+[role="_additional-resources"]
+== Additional resources
```

Comment:
So because this is an Additional resources section for a module and not for the entire assembly, you want to use `.` for the heading instead of `==`. Here's the [doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-assemblies) for more info on that: "Only use . formatting (.Additional resources) to follow a module in an assembly."

Using the `==` here treats this Additional resources section like its own discrete section, and in the right side navigation of the page, it looks like "Using skopeo to verify signatures of Red Hat container images" is a subsection of this Additional resources instead of "Understanding the verification of container images lacking verifiable signatures".

```suggestion
.Additional resources
```